src/autogluon/cloud/default_cluster_configs/ray_aws_default_cluster_config.yaml

```diff
@@ -2,12 +2,13 @@
 cluster_name: ag_ray_aws_default
 
 # The maximum number of workers nodes to launch in addition to the head
-max_workers: 2
+max_workers: 1
```

Comment:
why we reduce it to 1?

Reply:
This is just the default config, I set it to 1 to minimize cost. Though the default config will never be used actually because the cloud predictor will set the worker value to a custom one eventually